Question: Process to create a new release, any blockers?

So we're a makerspace with ~150 users and the v2.0.0 release crashes when trying to load the user database in the webui

I did download the head/main and build it manually and it works great!

Who would be incharge of testing and pushing this to a release so others could benefit from it? Are there any blockers for this?

I also wrote an entire n8n and nocodb management interface via MQTT that scales for multiple locks (which we need documentation for? cursor and the codebase was critical to me being able to do this) -- I'm happy to share that somewhere as it matures. We're in testing now

-Chris
Hive76 CTO

Build ended up in failure. 

```
In file included from .pio/libdeps/debug/AsyncTCP/src/AsyncTCP.cpp:4:0:
.pio/libdeps/debug/AsyncTCP/src/AsyncTCP.h:11:29: fatal error: esp_idf_version.h: No such file or directory
```

@k1n6b0b Did you encounter this error while building the head?

--- a/platformio.ini
+++ b/platformio.ini
@@ -11,7 +11,7 @@ upload_speed = 460800
 monitor_speed = 115200
 lib_deps =
   ArduinoJson@6.19.1
-  ESPAsyncTCP
+  me-no-dev/AsyncTCP@1.1.1
   ESPAsyncUDP
   https://github.com/lorol/ESPAsyncWebServer
   AsyncMqttClient@0.9.0

